rest/tests/test_misc.py

```diff
@@ -48,3 +53,41 @@ def test_root(self):
         self.assertEqual(HTTP_404_NOT_FOUND, resp.status_code)
         resp = self.client.options(API_BASE)
         self.assertEqual(HTTP_404_NOT_FOUND, resp.status_code)
+
+    def test_pagination(self):
+        """
+        Test page_size query parameter.
+        """
+        self.import_course_tarball(self.repo)
+
+        num_resources = LearningResource.objects.filter(
+            course__repository__id=self.repo.id
+        ).count()
+
+        def assert_page_size(page_size):
```

Comment:
nit: I would move this function at the beginning, after the doctoring of the parent function